INSTALL.md

```diff
@@ -118,6 +120,63 @@ If the desired python version is not installed, you can install it using
 [Xcode Command Line Tools](https://stackoverflow.com/questions/9329243/how-to-install-xcode-command-line-tools)
 needs to be installed for development.
 
+<a name="Apple-M1-Build-Dependencies"></a>
+##### Apple M1
+- Command line tools
+    ```bash
+    xcode-select --install
+    ```
+- If desire classical NEURON GUI<br>
+    From xquartz.org, click "Releases", click XQuartz-2.8.0_beta3 , and
+follow instructions. (after installing, logout and log back in)
+    If you desire single click button action for x11 when entering a window
+    ```bash
+    defaults write org.xquartz.X11 wm_ffm -bool true
+    ```
+   (For the new default to take effect, logout then log back in)
+- HomeBrew and Pip
+  ```bash
+  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+  echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> $HOME/.zprofile
+  eval $(/opt/homebrew/bin/brew shellenv)
+
+  brew install cmake
+  brew install open-mpi
+
+  pip3 install --user --upgrade pip
+  export PATH="$HOME/Library/Python/3.8/bin":$PATH
+  pip3 install --user cython
+  ```
+- Install
+  ```
+  cd $HOME
+  mkdir neuron
+  cd neuron
+  git clone https://github.com/neuronsimulator/nrn nrn
+  cd nrn
+  mkdir build
+  cd build
+  cmake .. -DCMAKE_INSTALL_PREFIX=install -DPYTHON_EXECUTABLE=`which python3` -DNRN_ENABLE_PYTHON_DYNAMIC=ON
+  #Note: without the dynamic python option, the build failed to find the library
```

Comment:
Under CMake I would check `CMAKE_OSX_ARCHITECTURES` (or something similar) if it's `arm64` then set default value of `NRN_ENABLE_PYTHON_DYNAMIC` to `ON`.

Reply:
Ok, And that will allow removal of this whole section. Strangely, to me, the Python executable is also the Python library.

Reply:
A cmake failure to find Python Lib I don't believe is specific to Apple M1.  The symptom is a cmake output of
```
-- Found PythonLibs: python3.8
...
--   LIB       |  python3.8
```
The reason that ```-DNRN_ENABLE_PYTHON_DYNAMIC=ON``` is a good work around is due to the fact that with that option there is no linking at build time to the python library.  The library is determined when ```nrniv``` is launched, by means of NRN_PYLIB determined by the nrnpyenv.sh bash script.  Perhaps it would be a good idea to incorporate this into the cmake build if cmake otherwise fails to find a valid python library.

It turned out that when I installed ```brew install gedit``` that a dependency was ```python3.9``` and if that is used, the cmake build is successful. (However a subsequent ```pip3 install --user numpy``` failed so I cannot get rxd to work. But that is another story).

Reply:
Hmm... I'm thinking that https://pypi.org/project/libpython/ is more principled than my nrnpyenv.sh script and therefore probably more robust in more environments.

Reply:
WIth the 5caf2ca change, ```cmake/modules/FindPythonLibsNew.cmake``` falls back last to ```find_libpython.py``` to find the path for ```PYTHON_LIB```. So on the Apple M1 with the default installed python3 (3.8), the ```-DNRN_ENABLE_PYTHON_DYNAMIC=ON``` is no longer needed.